templates/taxi/manufacturer_list.html

```diff
@@ -2,31 +2,33 @@
 
 {% block content %}
     <h1>Manufacturer List
+      <a href="{% url 'taxi:manufacturer-create' %}" style="color: blue"> + </a>
```

Comment:
Instead of using the plus symbol, try to use words to format it as it is better for UI.

Reply:
![image](https://github.com/mate-academy/py-taxi-service-forms/assets/168192382/621c353e-1d05-449c-8ee7-a393d34ee308)